src/Icons/.gitignore

```diff
@@ -1,4 +1,7 @@
-/vendor
+/assets/node_modules/
+/vendor/
 /composer.lock
-/var
+/phpunit.xml
 /.phpunit.result.cache
+
```

Comment:
```suggestion
```

Reply:
It was on purpose, common files across all components, and some specific files

Reply:
Could be done without empty space ;)